package-lock.json

```diff
@@ -3923,6 +4024,7 @@
       "version": "0.11.2",
       "resolved": "https://registry.npmjs.org/apollo-cache-control/-/apollo-cache-control-0.11.2.tgz",
       "integrity": "sha512-a1YdQLWzwuvZC7+xtJ8zno04pGRZG9595cX8W5YoXs2GeJqYdk+G3/tgdTgpLN827Vmgka0xkWfmxQUYm2xZUw==",
+      "dev": true,
```

Comment:
Granted I already said "entirety of this `package-lock.json`" in my above comment, this is _still_ unexpected given the changes in this change-set?

Reply:
Yeah, I think there was some good ol npm fun going on here.
6937cd8